protocols/weba.js

```diff
@@ -0,0 +1,120 @@
+const path = require('path');
+const fs = require('fs');
+const {fillTemplate, parseIdHash} = require('../util.js');
+
+const scnTemplateString = fs.readFileSync(path.join(__dirname, '..', 'type_templates', 'scn.js'), 'utf8');
+// const cwd = process.cwd();
+
+module.exports = {
+  resolveId(source, importer) {
+    return source;
+  },
+  async load(id) {
+    console.log('weba load id', {id});
+    id = id
+      .replace(/^(weba:\/(?!\/))/, '$1/');
+    
+    const match = id.match(/^weba:\/\/([0-9\.]+)\/([0-9\.]+)(?:\/|$)/i);
+    if (match) {
+      const x = parseFloat(match[1]);
+      const y = parseFloat(match[2]);
+      const coords = [x, y];
+
+      const sceneJson = {

```

Comment:
This can be broken out into the default scene itself. Totum protocol handlers should probably not be in the business of defining scene logic.